### ISSUE TITLE
if a sku contains space char it is incorrectly parsed by .sales() - urllib.parse.quote_plus(kwargs.pop('sku'))

### ISSUE THREAD

**Issue by LukasChristoph** [saleweaver/python-amazon-sp-api#890]
**Describe the bug**
If a SKU contains whitespace it wont get parsed corretly by urllib.parse.quote_plus(kwargs.pop('sku')) in Sales().
So you end up with the sales for the wrong sku and thats hard to detect. Because it returns correct response with 0 sales. 

**To Reproduce**
Call Sales with sku param.
Sku should contain a space char.

**Expected behavior**
Should request the sales for the correct sku.
Whitespace should get parsed correctly.

**Desktop (please complete the following information):**
Mac

**Additional context**
A sku with whitespace is very uncommen for us, so the bug was very hard to detect.

**BUGFIX**
urllib.parse.quote(kwargs.pop('sku')) parses the space char correctly and returns the correct sales data.
I didnt do further testing if .quote() instead of .quote_plus() breaks something else.

**Comment by saleweaver** [comment]
Yeah, it does break something else. I think this was fixed in another endpoint already, there were some issues Amazon side in the beginning...
